docs/sources/write/style-guide/style-conventions/index.md

```diff
@@ -219,3 +219,23 @@ On the published page, this warning renders as follows:
 By disabling authentication requirements, anyone can access your Grafana instance.
 There is a considerable security risk associated with this.
 {{% /admonition %}}
+
+## Semantic line breaks
+
+We use [these general rules](https://sembr.org/) for adding line breaks in your writing, but we recommend starting a new line after each sentence to make the text easier to review and edit.
```

Comment:
Given that we don't ourselves follow these rules in our documentation, may I suggest we soften this a bit:

```suggestion
The [Semantic Line Breaks organization](https://sembr.org/) suggests adding line breaks in your writing. We find this can help with reviews.

Note that we don't require that this suggestion be followed. We share the Semantic Line Breaks rules so that you can discuss with your review team and adapt to your circumstance.
```

Reply:
Fiona, I agree with your phrasing here very much. The phrase "[Grafana Writers' Toolkit] recommends" is a powerful oe, and as I think I've made clear we probably shouldn't be throwing our weight behind the SLB spec that we don't use in any of our docs or writer's toolkit. But I think it's good that we give people the option to use it whenever it can add clarity to the review process.